src/modules/commander/PreflightCheck.cpp

```diff
@@ -208,6 +208,29 @@ static bool imuConsistencyCheck(orb_advert_t *mavlink_log_pub, bool report_statu
 	return success;
 }
 
+static bool landConsistencyCheck(orb_advert_t *mavlink_log_pub, bool report_status)
+{
+	bool success = true; // start with a pass and change to a fail if any test fails
+
+	// Check the relationship between land detection threshold and land speed
+	float land_speed, land_detect_speed;
```

Comment:
If MC or VTOL

Reply:
That's true, but if that param is messed up on a fixed wing it would be equally disconcerting.